HeterogeneousCore/CUDAUtilities/test/OneToManyAssoc_t.h

```diff
@@ -0,0 +1,310 @@
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <random>
+#include <limits>
+#include <array>
+#include <memory>
+
+#ifdef __CUDACC__
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/requireDevices.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/currentDevice.h"
+
+#endif
+
+#include "HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h"
+
+using namespace cms::cuda;
```

Comment:
Code rule checker complains from `using namespace` in a header here as well. The nature of this file is really source, it just gets included in `.cc` and `.cu` files to get it built with `g++` and `nvcc`. It is also test code. I wonder if we should add an exception or figure out some other solution.

Reply:
We could rename it to `.impl` or something, but I would prefer not to, as it tend to loos syntax highlighting...